clean.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+from ndmtestpackage import clean
```

Comment:
probably this file is not required. We can invoke the clean() directly from the test.py as the last step.